### DISCUSSION TITLE
Patching synctex file failed errow when compiling a Rnw file.

### DISCUSSION THREAD

**Discussion by RaghuNaik** [WinEdt-Team/WinEdt-Support#24] [1.1 General Help and Discussion]
I always get this error window when I compile a Rnw document with Knitr. I click OK and things are fine. Not sure how to get rid of this error. I am running Winedt 11.1. Thanks.

![2024-03-19 11_49_45-Error](https://github.com/WinEdt-Team/WinEdt-Support/assets/6098486/d15a4942-0ef0-482f-8197-8422862b90ea)

**Comment by Alex-Simonic** [comment]
Knitr or R-compiler generates the error, and WinEdt cannot fix that. But it is easy to make WinEdt ignore this error without any warning. In %b\RManager\Exec you will find three macros:

Compile.edt, KnitrPDFTeXify.edt, and SweavePDFTeXify.edt

Open them in WinEdt. Around line 84 you will find:

```
  LetReg(0);
  IfFileExists('%b\_Err.log',!"ReadFile('%b\_Err.log',0);");
  IfStr('%!0','','<>',>
    !"Prompt('Patching synctex file failed!%\%\Error message:%\%!0',3,1);");
```
Replace the last line with:

```
    !"Relax;"); // !!!RaghuNaik: no prompt if it fails...
    //!"Prompt('Patching synctex file failed!%\%\Error message:%\%!0',3,1);");
```
The error is still there but WinEdt will not display any warning...

**Reply by RaghuNaik** [reply]
Thanks  Alex. That works.
